llvm/lib/MC/MCObjectFileInfo.cpp

```diff
@@ -343,7 +343,7 @@ void MCObjectFileInfo::initELFMCObjectFileInfo(const Triple &T, bool Large) {
   case Triple::mips64el:
     // We cannot use DW_EH_PE_sdata8 for the large PositionIndependent case
     // since there is no R_MIPS_PC64 relocation (only a 32-bit version).
-    if (PositionIndependent && !Large)
+    if (PositionIndependent)
```

Comment:
This just ignores the justification above?

Reply:
Yes. even with `Large` we still use sdata4, instead of sdata8.

```
For N64, maybe we should use sdata8, while GNU ld doesn't support it well, and in fact sdata4 is enough now.
```
Maybe, I will need to make it more clear.